cfn-publish.config

```diff
@@ -1,5 +1,5 @@
 bucket_name_prefix="solution-builders"
 acl="public-read"
 extra_files=build.zip
-templates="templates/template.yaml templates/vpc.yaml templates/role.yaml"
+templates="templates/template.yaml templates/role.yaml"
```

Comment:
I think it makes sense to stop publishing the VPC template since it's now "internal" to the solution but I'm open to suggestions

Reply:
I Agree.